docs/start/introduction.md

```diff
@@ -0,0 +1,84 @@
+| HPLC pump           | Component Specific | Specific access commands |
+| Pump                | Component Base     | Base access commands     |
+| Flowchem Component  | Class Base         | Base Constructor         |
+
+When constructing a class to represent a device, as in our case, ***Azura Compact Pump*** inherits the attributes and 
+methods of the class ***HPLC Pump***. This ***HPLC Pump*** class presents the specific standard commands for this pump 
+category. In turn, this class inherits the attributes and methods of the class ***Pump***. This class has the most 
+generic commands in all pumps (activate, infuse). This class still features the main class with attributes and methods 
+present on all devices. This level of abstraction allows the user to switch from one device to another very quickly, 
+as long as both devices have the same function (inherit the commands from ***HPLC Pump***, for example). It doesn't 
+matter if they come from suppliers and have different communication protocols. This device exchange will occur without
+changing anything in the already implemented code structure (a program related to the platform's automation). This 
+change will only occur in the configuration file, which is easily editable.
+
+## How easily editable is the configuration file?
+
+Once installed, flowchem can be used. The first step to using the flowchem is to write the configuration file. This file
+uses human-editable and friendly language. Each connected device must have a coding block, as shown below.
+
+```toml
+# Example of configuration flowchem file to access some devices
+[device.socl2]
+type = "Elite11"                 # Class name of the device
+port = "COM4"                    # Communication access
+syringe_diameter = "14.567 mm"   # Additional configuration
+syringe_volume = "10 ml"         # Additional configuration
+baudrate = 115200                # Additional configuration
+
+[device.hexyldecanoic]
+type = "AzuraCompact"          # Class name of the device
+ip_address = "192.168.1.119"   # Communication access
+max_pressure = "10 bar"        # Additional configuration
+
+[device.r4-heater]
+type = "R4Heater"              # Class name of the device
+port = "COM1"                  # Communication access
+
+[device.flowir]
+type = "IcIR"                                     # Class name of the device
+url = "opc.tcp://localhost:62552/iCOpcUaServer"   # Communication access
+template = "30sec_2days.iCIRTemplate"             # Additional configuration
+
+[device.s-pump]
+type = "ML600"             # Class name of the device
+port = "COM5"              # Communication port
+syringe_volume = "0.05 ml" # Additional configuration
+```
+Each block of code, separated by spacing, represents a device connected to the computer. The ***socl2***, ***hexyldecanoic***, 
+***r4-heater***, ***flowir***, ***s-pump*** are the names of the devices. Its name is flexible enough to be editable by 
```

Comment:
Might need the explain what is "Its name is flexible **enough** to be editable by its users. ?

Reply:
"separated by spacing" I think it is a little confusing.